BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/EditLineItem/Components/QuantityView.swift

```diff
@@ -46,7 +46,7 @@ final class QuantityView: UIView {
         button.setImage(prefferedImage(named: "quantity_minus_icon"), for: .normal)
         button.addTarget(self, action: #selector(decreaseQuantity), for: .touchUpInside)
         button.translatesAutoresizingMaskIntoConstraints = false
-        let descriptor = NSLocalizedStringPreferredFormat("ginibank.digitalinvoice.edit.minus.button.accessibility",
+        let descriptor = NSLocalizedStringPreferredGiniBankFormat("ginibank.digitalinvoice.edit.minus.button.accessibility",
                                                           comment: "Decrease quantity")
```

Comment:
[Suggestion] same suggestion: a indentation would be nice here, to align `comment` parameter in the right place